leadbutt.py

```diff
@@ -17,6 +19,8 @@
 import datetime
 import os.path
 import sys
+import time
+from retrying import retry
```

Comment:
Pardon the confusion, I closed this PR, because of the failed tests, if you could take a peek at #43, I'd be most appreciative.